src/conn/DBConnection.h

```diff
@@ -30,10 +30,10 @@ class DBConnection
     
     private:
         //Connection Variables
-        sql::Connection *conn;
-        sql::PreparedStatement *prep_stmt;
+        sql::Connection *_conn;
+        sql::PreparedStatement *_prep_stmt;
```

Comment:
Can we assume that _prepStmt and _conn are suitable shorthand?
Or do these need to be changed to _preparedStatement and _connection? 
(because "prettiness")